department-of-veterans-affairs/caseflow
# Show assignee name in success banner

Resolves #14217

### Description
Adds assignee name to success banner when assigning to an attorney. Also displays if this is an initial assignment or a reassignment

### Acceptance Criteria
- [ ] When assigning a case to an attorney, the assignee's name appears in the success banner
- [ ] When reassigning a case to an attorney, the assignee's name appears in the success banner and displays that this was a reassignment

### Testing Plan
1. Log in as BVAAABSHIRE and go to any appeal with a judge assign task
1. Assign the case to an attorney
1. Confirm atty name is in banner
1. Go back to case details and reassign the case to another attorney
1. Confirm "Reassigned" and attorney name appear in the banner
1. Log in as BVARDUNKLE
1. Go to any case that a judge has an assign task on
1. Assign the case to someone from the "other" dropdown
1. Confirm atty's name is in the success banner

### User Facing Changes

 BEFORE|REASSIGN|ASSIGN
 ---|---|---
![81324824-83780900-9065-11ea-9c58-e878d8bf7115](https://user-images.githubusercontent.com/45575454/83425565-e3ab6200-a3fb-11ea-80cf-c1593a0d5cc5.png)|![Screen Shot 2020-06-01 at 3 26 42 PM](https://user-images.githubusercontent.com/45575454/83446238-5f1d0b80-a41c-11ea-85d4-dfc2f3e81af0.png)|![Screen Shot 2020-06-01 at 3 27 00 PM](https://user-images.githubusercontent.com/45575454/83446247-617f6580-a41c-11ea-9167-e8e398299a3d.png)
